src/ice_masks.jl

```diff
@@ -187,11 +116,25 @@ function get_ice_masks(
             possible_ice_threshold=possible_ice_threshold,
         )
 
-        ice_mask[tile...] .= (morph_residue_seglabels .== floes_label)
+        ice_mask[tile...] .= (segmented_image_indexmap .== floes_label)
 
-        #  Conditionally update binarized_tiling as its not used in some workflows
-        binarize && (binarized_tiling[tile...] .= imbinarize(mrt))
     end
-
-    return (icemask=ice_mask, bin=binarized_tiling .> 0)
+    return ice_mask
 end
+
+# temp function until we replace tests to use IceDetectionThresholdMODIS721 directly
+function get_ice_labels_mask(ref_img::Matrix{RGB{N0f8}}, thresholds)
+    mask =
+        binarize(
+            ref_img,
+            IceDetectionThresholdMODIS721(;
+                band_7_max=thresholds[1],
+                band_2_min=thresholds[2],
+                band_1_min=thresholds[3]
+            ),
+        ) .|>
+        gray .|>
+        Bool
+    @debug "Found $(sum(mask)) ice pixels"
+    return mask
+end
```

Comment:
I'd suggest including a newline at the end of every file, just for consistency.
```suggestion
end

```